packages/frontend/src/lib/BootcStatusIcon.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
```

Comment:
Why have copies of upstream for star, spinner, but then not StatusIcon? Seems like we should just use it here.

Reply:
Reason is that the status that we use for history is different than the pods / containers / kubernetes concepts.

We need to have other statuses such as "error" or "lost" or "success" compared to StatusIcon which has:

```
$: solid = status === 'RUNNING' || status === 'STARTING' || status === 'USED' || status === 'DEGRADED';
```